cinema/urls.py

```diff
@@ -1,10 +1,28 @@
-from django.urls import path
+from django.urls import path, include
 
-from cinema.views import movie_list, movie_detail
+from rest_framework import routers
+
+from cinema.views import (
+    MovieViewSet,
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
```

Comment:
No need to include if basepath is not provided. Instead use:
urlpatterns = [list of urls] + router.urls